packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -109,6 +110,23 @@ function nodeTypeFromType(type) {
   return utilNodeTypeFromType(type);
 }
 
+// NOTE: should remove (1), (2), (3) after update new version of `react-is`
```

Comment:
this stuff is part of react-is@16.12 already. can we update it now?

Reply:
ok, I will update to `react-is@16.12` and update breaking changes